test-plone-4.3.x.cfg

```diff
@@ -10,3 +10,4 @@ test-extras = tests,tests_plone4
 # Downgrade collective.z3cform.datagridfield, since the Version 1.3.0 seems no
 # longer compatible with Plone 4.x
 collective.z3cform.datagridfield = 1.2
+six = 1.12.0
```

Comment:
I've added the missing dependency for six with a version constraint.
Overriding the version pinning is still necessary since Plone pins that to an older version.

Reply:
@buchi that means all 4.3 tests which upgrade to the new testbrowser will have to pin `six` up, right?
I think we should put the pinning in https://github.com/4teamwork/ftw-buildouts/blob/master/test-versions-plone-4.cfg then.

Reply:
I've added a pinning in https://github.com/4teamwork/ftw-buildouts/pull/171

Reply:
@buchi you can now remove that, since https://github.com/4teamwork/ftw-buildouts/pull/171 is merged. right?

Reply:
Removed pinning of six version.